exporter/awsxrayexporter/conn.go

```diff
@@ -53,6 +56,11 @@ const (
 	STSAwsCnPartitionIDSuffix = ".amazonaws.com.cn" // AWS China partition.
 )
 
+var collectorUserAgentHandler = request.NamedHandler{
+	Name: "otel.collector.UserAgentHandler",
+	Fn:   request.MakeAddToUserAgentHandler("opentelemetry-collector-contrib", version.Version, version.GitHash),
+}
```

Comment:
Version is passed to the components as part of the creation params in the factory method. You should use that instead of depending on the version module (see https://github.com/open-telemetry/opentelemetry-collector/blob/master/component/component.go#L123)

Reply:
Oops - thanks for pointing this out, fixed to use it